cmd/security-agent/app/app.go

```diff
@@ -115,12 +119,12 @@ func init() {
 }
 
 func newLogContext(logsConfig *config.LogsConfigKeys, endpointPrefix string) (*config.Endpoints, *client.DestinationsContext, error) {
```

Comment:
since the `newLogsContext` is called in two separate place, we probably need to pass the track-type in